bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -468,10 +476,15 @@ async def deactivate_infraction(
         return log_text
 
     @abstractmethod
-    async def _pardon_action(self, infraction: _utils.Infraction) -> t.Optional[t.Dict[str, str]]:
+    async def _pardon_action(
+        self,
+        infraction: _utils.Infraction,
+        notify: bool
```

Comment:
Could it be worth it to make this a kwarg like everywhere else you made this change?

Reply:
It could be done, but not really worth it IMO because in practice this won't get called directly with a True/False value — instead a variable is forwarded to it. This is the same reason it doesn't have a default value.